src/style.css

```diff
@@ -1 +1,88 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color-border: #e1e7ed;
+  --color-box-shadow-light: #3d4e611a;
+  --color-box-shadow-dark: #3d4e6133;
+  --color-text: #3d4e61;
+}
+
+.box__search1 {
+  margin-top: 20px;
+  display: flex;
+  border: 1px solid var(--color-border);
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 var(--color-box-shadow-light);
+}
+
+.box__search1:hover {
+  box-shadow: 0 3px 8px 0 var(--color-box-shadow-dark);
+  border: 1px solid var(--color-border);
+}
+
+.box__search2 {
+  margin-top: 20px;
+  display: flex;
+  border: 1px solid var(--color-border);
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 var(--color-box-shadow-light);
+}
+
+.box__search2:hover {
+  box-shadow: 0 3px 8px 0 var(--color-box-shadow-dark);
+  border: 1px solid var(--color-border);
+}
+
+.big__search {
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts. It's good practice to specify a generic family as a fallback in case the main font doesn't load. For example: `font-family: Avenir, Arial, sans-serif;`.